x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -35,6 +36,26 @@ func (k Keeper) SettlePendingClaims(ctx sdk.Context) (
 		return settledResult, expiredResult, err
 	}
 
+	// Capture the applications initial stake which will be used to calculate the
+	// max share any claim could receive from the application stake.
+	// This ensures that each claim can calculate the maximum amount it can take
+	// from an application's stake.
```

Comment:
Would you you taking another pass at this? I'm not sure if my suggestion aligns with what you were hoping to communicate:

```suggestion
	// Capture the applications initial stake which will be used to calculate the
	// max share any claim could burn from the application stake.
	// This ensures that each supplier can calculate the maximum amount it can take
	// from an application's stake.
```